react/components/MiniCartContent.js

```diff
@@ -40,13 +41,18 @@ class MiniCartContent extends Component {
     maxQuantity: MiniCartPropTypes.maxQuantity,
     showDiscount: MiniCartPropTypes.showDiscount,
     showShippingCost: MiniCartPropTypes.showShippingCost,
+
+    /* Update Items mutation */
+    updateItems: PropTypes.func.isRequired,
   }
 
   state = { isUpdating: [] }
 
-  sumItemsPrice = items => {
-    return items.reduce((sum, { listPrice, quantity }) => sum + listPrice * quantity, 0)
-  }
+  sumItemsPrice = items =>
```

Comment:
Are those items already filtered with `.filter(({ quantity }) => !!quantity)`? IDK if that check is for, for some reason, nil quantities or zeroed ones. If the latter, this is ok

Reply:
I don't think it's necessary. If there is no quantity, it's equal to zero. If it's equal to zero, why would I bother taking the item off the sum? It won't matter in the final result.